.github/workflows/build-test.yml

```diff
@@ -28,9 +28,6 @@ jobs:
       - name: Install packages
         run: yarn
 
-      - name: Check format
-        run: forge fmt --check
-
```

Comment:
why?

Reply:
we already have the separate format check